src/main/groovy/org/mockito/release/internal/gradle/ReleaseNotesPlugin.java

```diff
@@ -80,6 +80,7 @@ public void run() {
                         t.setGitHubReadOnlyAuthToken(conf.getGitHub().getReadOnlyAuthToken());
                         t.setGitHubRepository(conf.getGitHub().getRepository());
                         t.setPreviousVersion(conf.getPreviousReleaseVersion());
+                        t.setSkipCommitMessagePostfix(conf.getGit().getCommitMessagePostfix());
```

Comment:
Hmmm, I think this will not work. The design I suggested earlier has a flaw. The commitMessagePostfix is different with each release build because it includes Travis build number.

I pushed a meaty TODO to this branch with suggested update to the design. Hopefully, it will work better!

Thank you @NagRock for working on this! Removing hardcoded values and making the library more flexible will work in our favor and help getting wider adoption!